sdk/typescript/src/cryptography/ed25519-keypair.ts

```diff
@@ -140,4 +146,12 @@ export class Ed25519Keypair implements Keypair {
 
     return new Ed25519Keypair({ publicKey: pubkey, secretKey: fullPrivateKey });
   }
+
+  export(): ExportedKeypair {
```

Comment:
can this just reuse the `Keypair` type?

Reply:
Which one do you mean? `sdk/typescript/src/cryptography/keypair.ts`?

Reply:
yes

Reply:
I don't see how it can be reused here, this is a different type. This is (will be) just the private key and the schema but `Keypair` has other stuff as well. Could you explain what you mean?

Reply:
Do you think renaming the function to serialise and deserialise would be more clear?

Reply:
i see, so exportedKeyPair is just a serialization of the key pair